### PR TITLE
Adding instructions for contributing to the bibliography including usage of Zotero

### LINKED ISSUE #1458: Documentation on how to obtain bibtex entries with Zotero
**Description**
Something along these lines should appear in the documentation:
1.  Get https://www.zotero.org/
2. Get better BibTeX https://retorque.re/zotero-better-bibtex/installation/index.html
3.  Use auth.lower + year + shorttitle(1,0) for citation key generation (old entries should be pinned)
4. Get an account
5. Add this group https://www.zotero.org/groups/5371154/rdmkit
6. Add/Import items to zotero (optional to group above)
7. Export as better BibTex and append to references.bib

### PR DESCRIPTION
**Pull request by bianchini88**
Closes #1458

### CODE REVIEW

**Comment by bianchini88** [comment]
@korbinib @martin-nc @bedroesb A first attempt at providing more detailed documentation about adding items to the bibliography. I will be happy to receive feedback from you.

**Review by korbinib — APPROVED** [review]
Looks good for me. Thank you for writing this up!

File: _data/tool_and_resource_list.yml [hunk 1]
```diff
@@ -2634,3 +2634,11 @@
   registry:
     biotools: hpo
   url: https://hpo.jax.org/app/
+- description: free and open-source reference management software to manage bibliographic data and related research materials
```
Comment (martin-nc) [inline _data/tool_and_resource_list.yml:2637]:
```suggestion
- description: free and open-source reference management software to manage bibliographic data and related research materials
```
Add a capital for "free" and a full stop at the end? So: "Free and open-source reference management software to manage bibliographic data and related research materials."

File: _data/tool_and_resource_list.yml [hunk 2]
```diff
@@ -2634,3 +2634,11 @@
...
+  id: zotero
+  name: Zotero
+  url: https://www.zotero.org/
+- description: a plugin for Zotero and Juris-M that makes it easier to manage bibliographic data
```
Comment (martin-nc) [inline _data/tool_and_resource_list.yml:2641]:
```suggestion
- description: A plugin for Zotero and Juris-M that makes it easier to manage bibliographic data.
```
Same again.

File: pages/contribute/style_guide.md [hunk 3]
```diff
@@ -50,9 +50,40 @@ In general, we follow the European Commission's [Web Writing Style Guide](https:
   * **Training:** training is an uncountable noun and cannot have a plural. You can write "training courses" and "training materials" but not "trainings".
 
 ## Bibliography
- * Add your citations as bibtex to the `_bibliography/references.bib` file.
- * Add {% raw %} `{% cite reference_key %}` {% endraw %} to the text where you are citing the reference.
- * Add {% raw %} `## Bibliography {% bibliography --cited %}` {% endraw %} to show a bibliography section containing the references in the page.
+ * RDMkit uses BibTeX as a tool for creating bibliography.
```
Comment (martin-nc) [inline pages/contribute/style_guide.md:53]:
```suggestion
 * RDMkit uses BibTeX as a tool for creating a bibliography.
```

File: pages/contribute/style_guide.md [hunk 4]
```diff
@@ -50,9 +50,40 @@ In general, we follow the European Commission's [Web Writing Style Guide](https:
...
+  year = {2024},
+  number = {1},
+  doi = {DOI},
+  langid = {english}
+}
+```
+Make sure that the *author* field does not contain *et al.*, as this substitution is performed via the class and should
+not be part of the information provided by the contributors.
+
+* If you are a {% tool "zotero" %} user, we recommed
```
Comment (martin-nc) [inline pages/contribute/style_guide.md:78]:
```suggestion
* If you are a {% tool "zotero" %} user, we recommend that you:
```

File: pages/contribute/style_guide.md [hunk 5]
```diff
@@ -50,9 +50,40 @@ In general, we follow the European Commission's [Web Writing Style Guide](https:
...
- * Add your citations as bibtex to the `_bibliography/references.bib` file.
- * Add {% raw %} `{% cite reference_key %}` {% endraw %} to the text where you are citing the reference.
- * Add {% raw %} `## Bibliography {% bibliography --cited %}` {% endraw %} to show a bibliography section containing the references in the page.
+ * RDMkit uses BibTeX as a tool for creating bibliography.
+ * Follow the following instuctions to create and display a bibiography on your page:
+   1. Add your citations in the `_bibliography/references.bib` file. Refer to the instructions below for more information on the format.
+   1. Add {% raw %} `{% cite reference_key %}` {% endraw %} to the text where you citing one of your entries.
+   1. Add {% raw %} `## Bibliography {% bibliography --cited %}` {% endraw %} in your page to show a bibliography section containing the references you have added following the instructions in the previous point. 
+* We use a display class based on *Nature* publications. Your BibTeX files should follow a standardised format described below.
+  We recommend using {% tool "zotero" %} to automatise the process (see below).
```
Comment (martin-nc) [inline pages/contribute/style_guide.md:59]:
```suggestion
  We recommend using {% tool "zotero" %} to automate the process (see below).
```

File: pages/contribute/style_guide.md [hunk 6]
```diff
@@ -50,9 +50,40 @@ In general, we follow the European Commission's [Web Writing Style Guide](https:
...
+  doi = {DOI},
+  langid = {english}
+}
+```
+Make sure that the *author* field does not contain *et al.*, as this substitution is performed via the class and should
+not be part of the information provided by the contributors.
+
+* If you are a {% tool "zotero" %} user, we recommed
+  1. Install the {% tool "better-bibtex" %} plugin.
+  1. On your desktop version of {% tool "zotero" %}, open the *Better BibTeX preferences* (under *Tools > Better BibTeX*).
```
Comment (martin-nc) [inline pages/contribute/style_guide.md:80]:
```suggestion
  2. On your desktop version of {% tool "zotero" %}, open the *Better BibTeX preferences* (under *Tools > Better BibTeX*).
```
All the numbers are "1." - is that intentional?

File: pages/contribute/style_guide.md [hunk 7]
```diff
@@ -50,9 +50,40 @@ In general, we follow the European Commission's [Web Writing Style Guide](https:
...
+  in the *citation keys* tab, edit the *citation key formula* to `auth.lower + year + shorttitle(1,0)`
```
Comment (martin-nc) [inline pages/contribute/style_guide.md:81]:
```suggestion
  in the *citation keys* tab, change the *citation key formula* to `auth.lower + year + shorttitle(1,0)`
```
?

File: pages/contribute/style_guide.md [hunk 8]
```diff
@@ -50,9 +50,40 @@ In general, we follow the European Commission's [Web Writing Style Guide](https:
...
+  1. Import the file e.g. using the *Add item by identifier* functionality.
```
Comment (martin-nc) [inline pages/contribute/style_guide.md:82]:
```suggestion
  1. Import the file e.g. using *Add item by identifier*.
```

**Review by martin-nc — COMMENTED** [review]
@bianchini88 I just made a few comments on the text. Are they okay?

**Comment by bedroesb** [comment]
We also might have to add a page_id to this page to get the tools table at the bottom working!

File: pages/contribute/style_guide.md (hunk 6 above)
Comment (bianchini88) [reply pages/contribute/style_guide.md:80]: That's what we recommend to do [here](https://github.com/elixir-europe/rdmkit/blob/a716351446f11993989b3d948cde78714decb8c4/pages/contribute/markdown_cheat_sheet.md), I think.

**Comment by bianchini88** [comment]
Thank you @martin-nc!

**Review by martin-nc — APPROVED** [review]
Looks okay now!

**Comment by bedroesb** [comment]
Are we ok with having no toolstable at the bottom of the styleguide, since it is not part of the "Data management" pages?

Also no related page is currently linked to the tools because of that: 
image


This because it needs a page_id

**Comment by bedroesb** [comment]
For me it is both fine, just wanted to make sure we are aware :)

**Comment by martin-nc** [comment]
> Are we ok with having no toolstable at the bottom of the styleguide

Yes, I don't think it needs to be there.

**Comment by bianchini88** [comment]
> Yes, I don't think it needs to be there.

Agreed!

**Review by bedroesb — APPROVED** [review]
Than it all looks good!

**Pull request merged**